modals/src/main/java/com/decathlon/vitamin/compose/modals/impl/VitaminModalImpl.kt

```diff
@@ -23,22 +23,22 @@ internal fun VitaminModalImpl(
     title: String?,
     painter: Painter?,
     contentScrollState: ScrollState,
-    positiveButton: (@Composable () -> Unit)?,
-    negativeButton: (@Composable () -> Unit)?,
-    neutralButton: (@Composable () -> Unit)?,
+    rightButton: (@Composable () -> Unit)?,
```

Comment:
just `rightButton` don't give me the information that it is required

Reply:
maybe add a comment on the `rightButton != null` condition in the implementation